time.ZonedDateTime.parse and format fails with "use locale" error

Users will frequently come into a case where they want to convert a DateTime string to a ZonedDateTime. There is a powerful parsing capability built into time.ZonedDateTime but when trying to use it I get:

```
2022-01-03 10:46:41.247 [ERROR] [internal.handler.ScriptActionHandler] - Script execution of rule with UID 'dads_motion' failed: org.graalvm.polyglot.PolyglotException: JsJodaException: Pattern using (localized) text not implemented, use @js-joda/locale plugin!
```

So I've tried adding to the top of my script actions:

    require('@js-joda/locale_en_us');
    require('js-joda/locale_en_us');
    require('locale_en_us');
    require('@locale_en_us');
    require('time.locale_en_us');
    require('@time.locale_en_us');

All generate a "Cannot load CommonJS module"
```
2022-01-03 10:49:25.168 [ERROR] [internal.handler.ScriptActionHandler] - Script execution of rule with UID 'dads_motion' failed: org.graalvm.polyglot.PolyglotException: TypeError: Cannot load CommonJS module
: 'js-joda/locale_en_us'
```

Others have reported the same problem with `locale_de` as well. As far as I can tell, at this point it's impossible to parse or format a date time string in a rule using time.ZonedDateTime without the locale imported. But I can't figure out a way to import it.

It seems like it should be imported already but can see how that might be a problem given people will want a different locale based on where they are. So I would expect that we be able to import the locale plugin as it's described in the error message, and the joda-js docs, or it be available to import through the `time` namespace or `utils` namespace. 

With the information I have right now I'm stuck.

As an aside, is it possible to use the joda-js Classes with the Java equivalents? For example, can I pass a Java ZonedDateTime to a joda-js Duration? (I know, I'll try it later but if someone knows off the top of their head...). If not, we may need to provide a function to normalize/convert from the Java to the joda-js which *should* be straight forward except for the fact that parsing Strings isn't working.

--- a/README.md
+++ b/README.md
@@ -833,7 +833,7 @@ See [JS-Joda](https://js-joda.github.io/js-joda/) for more examples and complete
 
 Occasionally, one will need to parse a non-supported date time string or generate one from a ZonedDateTime.
 To do this you will use [JS-Joda DateTimeFormatter and potentially your Locale](https://js-joda.github.io/js-joda/manual/formatting.html).
-However, all the locales combine to a unacceptibly large size to ship them with the openhab-js library.
+However, shipping all the locales with the openhab-js library would lead to an unacceptable large size.
 Therefore if you attempt to use the `DateTimeFormatter` and receive an error saying it cannot find your locale, you will need to manually install your locale and import it into your rule.
 
 [JS-Joda Locales](https://github.com/js-joda/js-joda/tree/master/packages/locale#use-prebuilt-locale-packages) includes a list of all the supported locales.

--- a/README.md
+++ b/README.md
@@ -829,6 +829,23 @@ actions.Exec.executeCommandLine(time.Duration.ofSeconds(20), 'echo', 'Hello Worl
 
 See [JS-Joda](https://js-joda.github.io/js-joda/) for more examples and complete API usage.
 
+### Parsing and Formatting
+Occasionally, one will need to parse a non-supported date time string or generate one from a ZonedDateTime.
+To do this you will use [JS-Joda DateTimeFormatter and potentially your Locale](https://js-joda.github.io/js-joda/manual/formatting.html).
+However, all the locales combine to a unacceptibly large size to ship them with the openhab-js library.
+Therefore if you attempt to use the `DateTimeFormatter` and receive an error saying it cannot find your locale, you will need to install your locale and import it into your rule.
+
+[JS-Joda Locales](https://github.com/js-joda/js-joda/tree/master/packages/locale#use-prebuilt-locale-packages) includes a list of all the supported locales.
+Each loacle consists of a two letter language indicator followed by a "-" and a two letter dialect indicator: e.g. "EN-US".
+Installing a locale can be done through the command `npm install @js-joda/locale_de-de` from the *$OPENHAB_CONF/automation/js* folder.
+
+To import and use a local into your rule you need to require it and create a `DateTimeFormatter` that uses it.
+
+```javascript
+var Locale = require('@js-joda/locale_de-de').Locale.GERMAN;
+var formatter = time.DateTimeFormatter.ofPattern('MM/dd/yy HH:mm').withLocale(Locale);
+```
+
 #### `time.toZDT()`
 
 There will be times when this automatic conversion is not available (for example when working with date times within a rule).

--- a/README.md
+++ b/README.md
@@ -830,6 +830,7 @@ actions.Exec.executeCommandLine(time.Duration.ofSeconds(20), 'echo', 'Hello Worl
 See [JS-Joda](https://js-joda.github.io/js-joda/) for more examples and complete API usage.
 
 ### Parsing and Formatting
+
 Occasionally, one will need to parse a non-supported date time string or generate one from a ZonedDateTime.
 To do this you will use [JS-Joda DateTimeFormatter and potentially your Locale](https://js-joda.github.io/js-joda/manual/formatting.html).
 However, all the locales combine to a unacceptibly large size to ship them with the openhab-js library.

--- a/README.md
+++ b/README.md
@@ -829,7 +829,7 @@ actions.Exec.executeCommandLine(time.Duration.ofSeconds(20), 'echo', 'Hello Worl
 
 See [JS-Joda](https://js-joda.github.io/js-joda/) for more examples and complete API usage.
 
-### Parsing and Formatting
+#### Parsing and Formatting
 
 Occasionally, one will need to parse a non-supported date time string or generate one from a ZonedDateTime.
 To do this you will use [JS-Joda DateTimeFormatter and potentially your Locale](https://js-joda.github.io/js-joda/manual/formatting.html).

--- a/README.md
+++ b/README.md
@@ -844,7 +844,7 @@ To import and use a local into your rule you need to require it and create a `Da
 
 ```javascript
 var Locale = require('@js-joda/locale_de-de').Locale.GERMAN;
-var formatter = time.DateTimeFormatter.ofPattern('MM/dd/yy HH:mm').withLocale(Locale);
+var formatter = time.DateTimeFormatter.ofPattern('dd.MM.yyyy HH:mm').withLocale(Locale);
 ```
 
 #### `time.toZDT()`

--- a/README.md
+++ b/README.md
@@ -834,7 +834,7 @@ See [JS-Joda](https://js-joda.github.io/js-joda/) for more examples and complete
 Occasionally, one will need to parse a non-supported date time string or generate one from a ZonedDateTime.
 To do this you will use [JS-Joda DateTimeFormatter and potentially your Locale](https://js-joda.github.io/js-joda/manual/formatting.html).
 However, all the locales combine to a unacceptibly large size to ship them with the openhab-js library.
-Therefore if you attempt to use the `DateTimeFormatter` and receive an error saying it cannot find your locale, you will need to install your locale and import it into your rule.
+Therefore if you attempt to use the `DateTimeFormatter` and receive an error saying it cannot find your locale, you will need to manually install your locale and import it into your rule.
 
 [JS-Joda Locales](https://github.com/js-joda/js-joda/tree/master/packages/locale#use-prebuilt-locale-packages) includes a list of all the supported locales.
 Each loacle consists of a two letter language indicator followed by a "-" and a two letter dialect indicator: e.g. "EN-US".

--- a/README.md
+++ b/README.md
@@ -840,7 +840,7 @@ Therefore if you attempt to use the `DateTimeFormatter` and receive an error say
 Each loacle consists of a two letter language indicator followed by a "-" and a two letter dialect indicator: e.g. "EN-US".
 Installing a locale can be done through the command `npm install @js-joda/locale_de-de` from the *$OPENHAB_CONF/automation/js* folder.
 
-To import and use a local into your rule you need to require it and create a `DateTimeFormatter` that uses it.
+To import and use a local into your rule you need to require it and create a `DateTimeFormatter` that uses it:
 
 ```javascript
 var Locale = require('@js-joda/locale_de-de').Locale.GERMAN;

